tests/test_loss/test_classification_loss.py

```diff
@@ -37,3 +45,8 @@ def test_bce_loss():
     assert torch.allclose(
         loss(fake_pred, fake_label, fake_weight),
         -torch.log(torch.tensor(0.5)))
+
+
+if __name__ == '__main__':
```

Comment:
We do not need `__main__` in pytest

Reply:
ok. I use this to test locally. I forget to delete it when I commit.

Reply:
You can run pytest locally without __main__.

Reply:
ok!